File: Python3/1.5/dicegame.py
Gunnar = g1, g2, g3, g4 = [int(x) for x in input().split()]
Emma = e1, e2, e3, e4 = [int(x) for x in input().split()]
if (sum(Emma)/4>sum(Gunnar)/4):
	print("Emma")
elif (sum(Emma)/4<sum(Gunnar)/4):
	print("Gunnar")
else:
	print("Tie")

File: 1.5/Dicegame/Dicegame.cpp
#include <iostream>
using namespace std;
int main()
{
    int g1, g2, g3, g4;
    int e1, e2, e3, e4;
    cin >> g1 >> g2 >> g3 >> g4;
    cin >> e1 >> e2 >> e3 >> e4;
    double gunnar = g1 + g2 + g3 + g4;
    double emma = e1 + e2 + e3 + e4;
    if (gunnar > emma) {
        cout << "Gunnar";
    }
    else if (emma > gunnar) {
        cout << "Emma";
    }
    else {
        cout << "Tie";
    }
    return 0;
}